src/scenes/PeopleRouter/PeopleListView.tsx

```diff
@@ -399,6 +399,7 @@ export default function PeopleListView(): JSX.Element {
                       rows={results}
                       orderBy='name'
                       Renderer={TableCellRenderer}
+                      isClickable={() => false}
```

Comment:
Should it be clickable for admins?

Reply:
Admins still have the checkbox to select it.  I don't think we need selectable rows if we have the checkboxes.